sparta/example/CoreModel/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ sparta_application(sparta_core_example)
 # Add the custom regress/regress_valgrind targets.
 add_custom_target (core_example_regress)
 add_custom_target (core_example_regress_valgrind)
+set (VALGRIND_TEST_LABEL valgrind_test)
```

Comment:
NBD, but If it works for you, could we just set the label once up in the common example/CMakeLists.txt

Reply:
Yep

Reply:
Done.